app/assets/javascripts/course_users.js

```diff
@@ -49,6 +49,10 @@ const validateIdentity = (input) => {
 };
 
 document.addEventListener('DOMContentLoaded', function () {
+  $('#lti-settings-modal').modal({
+    dismissible: false
```

Comment:
Perhaps add `preventScrolling: false,`

Currently, the scrollbar disappears when you open the modal, which causes the background to shift. Setting this option preserves the scrollbar (and allows the user to scroll with the modal open, which I think is okay).

Ref: https://materializecss.com/modals.html

Reply:
Oh nice, will add.